netbox/dcim/migrations/0154_half_height_rack_units.py

```diff
@@ -18,6 +18,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='device',
             name='position',
-            field=models.DecimalField(blank=True, decimal_places=1, max_digits=4, null=True, validators=[django.core.validators.MinValueValidator(1), django.core.validators.MaxValueValidator(99.5)]),
+            field=models.DecimalField(blank=True, decimal_places=1, max_digits=4, null=True, validators=[django.core.validators.MinValueValidator(1), django.core.validators.MaxValueValidator(100.5)]),
```

Comment:
We should probably have this defined as a constant, it is sort of a magic number right now.  Also wondering in regards to https://github.com/netbox-community/netbox/pull/12778 would people potentially want racks to go beyond this (i.e. start at 200 up to 300?)

Reply:
We only support rack heights up to 100, which already exceeds any practical real-world instance. If anyone voices a need for larger racks it's almost certainly because they're misusing the model.

I'll define a constant for the upper bound, though I doubt it would have mitigated this bug anyway as it's a classic off-by-one error.